### PR TITLE
Add systemd layer

### PR DESCRIPTION
**Pull request by StreakyCobra**

### CODE REVIEW

File: layers/+config-files/systemd/README.org [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#+begin_src emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(systemd))
+#+end_src
+
+* Key bindings
+
+| Key Binding   | Description                                                              |
+|---------------+--------------------------------------------------------------------------|
+| ~<SPC> m h d~ | Systemd open the directives documentation                                |
+| ~<SPC> m h o~ | Systemd open documentation (if a =\=Documentation= directive is provided |
```
Comment (jmiven) [inline layers/+config-files/systemd/README.org]: There is a formatting problem and a parenthesis is missing.

**Comment by jmiven** [comment]
:+1: Thanks!

**Comment by StreakyCobra** [comment]
@jmiven Updated, thx :+1:

**Comment by syl20bnr** [comment]
Thank you ! :+1: 
It is possible that @jmiven comment has not been cherry-picked.
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**